s3_transport/include/irods/private/s3_transport/s3_transport.hpp

```diff
@@ -552,12 +553,12 @@ namespace irods::experimental::io::s3_transport
 
                 return shm_obj.atomic_exec([this, _buffer, _buffer_size](auto& data) {
 
-                    auto position_before_write = this->cache_fstream_.tellp();
+                    std::streamoff position_before_write = this->cache_fstream_.tellp();
```

Comment:
The compiler didn't like the type deduced through the use of `auto`? What is the deduced type?

Reply:
The compiler liked it fine. fmtlib didn't. I don't remember what it was before.

Reply:
How did you land on `std::streamoff`? Is that the type returned by `.tellp()`?

Reply:
IIRC, it's an implicit cast from `.tellp()`'s return type, and I landed on it because I saw we were passing in `std::streamoff` somewhere else without issue.